purchase_reception_status_line/models/purchase_order_line.py

```diff
@@ -54,15 +48,21 @@ def _compute_reception_status(self):
             if line.order_id.state in ("purchase", "done"):
                 if line.force_received:
                     status = "received"
-                elif (
-                    float_compare(
-                        line.qty_received, line.product_qty, precision_digits=prec
-                    )
-                    >= 0
-                ):
-                    status = "received"
-                elif float_compare(line.qty_received, 0, precision_digits=prec) > 0:
-                    status = "partial"
-            line.reception_status = (
-                status if not line.order_id.force_received else "received"
-            )
+                else:
+                    if (
+                        float_compare(
+                            line.qty_received, line.product_qty, precision_digits=prec
+                        )
+                        > 0
+                    ):
+                        status = "over"
+                    elif (
+                        float_compare(
+                            line.qty_received, line.product_qty, precision_digits=prec
+                        )
+                        >= 0
```

Comment:
```suggestion
                        == 0
```
if it's greater than 0, it will already fall in the first if statement